openlibrary/catalog/add_book/__init__.py

```diff
@@ -568,4 +628,7 @@ def load(rec):
             assert isinstance(i, dict)
 
         web.ctx.site.save_many(edits, 'import new book')
+
+    # update_ia_metadata_for_ol_edition(reply['edition']['key'].split('/')[2])
```

Comment:
Should this comment stay?

Reply:
placeholder next stage where we have importbot and addwork write back to IA automatically